src/System.Runtime.Serialization.Primitives/src/Configurations.props

```diff
@@ -4,7 +4,7 @@
     <BuildConfigurations>
       netcore50aot-Windows_NT;
       uap101aot-Windows_NT;
-      net463-Windows_NT;
+      netfx-Windows_NT;
```

Comment:
Because I want to be sure that they only build for the windows build virtual. netfx-Unix makes really no sense and so if folks want to target netfx then they should either be on windows or specify OSGroup=Windows_NT.